src/main.rs

```diff
@@ -214,6 +214,22 @@ fn tray_icon_name() -> String {
 /*******************************************************************************************************************/
 /* Clipboard functions for different OSes */
 
+#[cfg(target_os = "linux")]
+fn copy_to_clipboard(s: &str) {
+    let _ = Command::new("/usr/bin/xclip").stdin(Stdio::piped()).stdout(Stdio::inherit()).stderr(Stdio::inherit()).spawn().map(|mut c| {
+        c.stdin.take().map(|mut stdin| {
+            let _ = stdin.write_all(s.as_bytes());
+        });
+        let _ = c.wait();
+    });
+}
+
+#[cfg(target_os = "linux")]
+fn read_from_clipboard() -> String {
+    Command::new("/usr/bin/xclip").output().map_or_else(|_| String::new(), |out| String::from_utf8(out.stdout).map_or_else(|_| String::new(), |s| s))
+}
+
+#[cfg(windows)]
```

Comment:
Should this be here?

Reply:
I think your question is related to the issue (https://github.com/zerotier/DesktopUI/issues/8) you opened for Wayland support? We probably can use a rust library for cross-platform copy paste including Wayland support. I will mention that in the opened issue.

Or did you mean something else?

Reply:
> Or did you mean something else?

I meant the above line specifically `#[cfg(windows)]`, it seems potentially out of place/like an accidental inclusion?